### ISSUE TITLE
[CoE Starter Kit - BUG] CoE flows are triggering activity on inactive environments. 

### ISSUE THREAD

**Issue by oburgoscr** [microsoft/coe-starter-kit#9085]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

We have several Teams and Developer environments that are inactive. We know nobody has been using them for more than 6 months and we have confirmed this information in the CoE tables. However, the automatic cleanup process is not deleting these inactive environments. We opened a ticket with Microsoft and they shared a list of CoE flows they think are causing this activity.

Admin | Sync Template v4 (Portals)
CLEANUP Helper - Check Deleted v4 (Portals)
CLEANUP Helper - Check Deleted v4 (Model Driven Apps)
Admin | Sync Template v4 (Desktop Flow - Runs)
Admin | Sync Template v4 (Driver)
Admin | Sync Template v4 (Ai Models)
Admin | Sync Template v4 (Desktop Flows)
Admin | Sync Template v4 (Model Driven Apps)
CLEANUP HELPER - Check Deleted v4 (Desktop Flows)
CLEANUP Helper - Check Deleted v4 (PVA)
CLEANUP Helper - Check Deleted v4 (Solutions)
CLEANUP HELPER - Check Deleted v4 (Business Process Flows)
Admin | Sync Template v4 (Solutions)
Admin | Sync Template v4 (PVA)
Admin | Sync Template v4 (Business Process Flows)
CLEANUP HELPER - Check Deleted v4 (Ai Models)

### Expected Behavior

Inactive environments should get marked as inactive and eventually deleted automatically by the cleanup process.

[Automatic deletion of inactive Dataverse for Teams environments - Power Platform | Microsoft Learn]
(https://learn.microsoft.com/en-us/power-platform/admin/inactive-teams-environment). 

[Automatic deletion of Power Platform environments - Power Platform | Microsoft Learn]
(https://learn.microsoft.com/en-us/power-platform/admin/automatic-environment-cleanup).

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.37

### What app or flow are you having the issue with?

Admin | Sync Template v4 (Solutions)

### What method are you using to get inventory and telemetry?

Cloud flows

**Comment by Jenefer-Monroe** [comment]
Hello. @Grant-Archibald-MS is driving this with the product team. It is currently the expected product behavior from envt reads
Please see https://github.com/microsoft/coe-starter-kit/issues/7574

**Comment by oburgoscr** [comment]
Hi Jenefer, thanks for looking into this issue.

Last year, we opened a support ticket with Microsoft and they were able to resolve the issue. I believe the Product Team was involved in the solution, but I’m not sure if it was properly documented.

Now, the issue has resurfaced (I'm not sure if it was after one of our CoE upgrades), and although I opened a new support ticket, the agent is having trouble locating the previous solution, even though I shared the resolved ticket number. Would it be helpful for your team to have that ticket number?

If so, what would be the best and most secure way to share it with you?
